README.md

```diff
@@ -224,9 +224,9 @@ bool onWatchList = amlResult.IsOnWatchList();
 1) Open the Example.sln solution in Visual Studio, found in the [/src/Examples](/src/Examples) folder
 1) Rename the [secrets.config.example](src/Examples/45Example/secrets.config.example) file to `secrets.config`
 1) Fill in the environment variables in this file with the ones specific to your application (mentioned in the [Client initialisation](#client-initialisation) section)
-1) From the Yoti Dashboard, set the Callback URL of your application to be `localhost:55555/account/connect`. You'll need to change it from `https` to `http` once the webpage loads, as the Dashboard doesn't allow `http` URLs
+1) From the Yoti Dashboard, set the Callback URL of your application to be `localhost:44321/account/connect`.
```

Comment:
... set the application domain to `localhost:44321`  and the scenario callback URL to `account/connect`